packages/help-center/src/components/utils.tsx

```diff
@@ -14,3 +15,45 @@ export const generateContactOnClickEvent = (
 		} );
 	}
 };
+
+export const getFilteredConversations = ( {
+	conversations,
+}: {
+	conversations: ZendeskConversation[];
+} ) => {
+	const recentConversations: ZendeskConversation[] = [];
+	const archivedConversations: ZendeskConversation[] = [];
+
+	if ( ! conversations ) {
+		return {
+			recentConversations,
+			archivedConversations,
+		};
+	}
```

Comment:
Do we need this check? If conversations is null it is not gonna cycle in `conversations.forEach` and it is going to return anyways at the end of the function!

Reply:
This is not needed. I removed this condition.